content/webapp/types/exhibitions.ts

```diff
@@ -52,6 +52,8 @@ export type Exhibition = GenericContentFields & {
   relatedIds: string[];
   seasons: Season[];
   contributors: Contributor[];
+  accessResourcesPdfs: (Link & { size: number })[];
+  accessResourcesText?: prismic.RichTextField;
```

Comment:
mega detail but could these be grouped under the same root as they are related? (I think)

Reply:
possibly, but I don't think it's really worth changing unless you feel strongly about it

Reply:
I don't!